scripts/run-gtest.ps1

```diff
@@ -660,6 +662,8 @@ function Wait-TestCase($TestCase) {
                 LogErr "$($TestCase.Name) failed (in $($Delta.TotalSeconds) sec):"
                 if ($StdOutTxt) { Write-Host $StdOutTxt }
                 if ($StdErrorTxt) { Write-Host $StdErrorTxt }
+            } elseif($AnyTestSkipped) {
+                Log "SKIPPED $($TestCase.Name) (in $($Delta.TotalSeconds) sec)"
```

Comment:
```suggestion
                Log "$($TestCase.Name) skipped"
```
Let's follow the same format and keep test name first. I also don't see a reason to log the time.

Reply:
I rearranged it to make the skipped tests stand out in the thousands of tests that we run. The timestamp is of low utility, so I can remove it

Reply:
Bottom line, I prefer the consistency.